src/extension/background/index.ts

```diff
@@ -7,17 +7,6 @@ import type {
   MessagePanel,
 } from '../../types/shared';
 
-/* ------------------------------------------------------ */
-/**
- * This is used to keep the background script alive. More testing is needed to determine whether it is needed or not
- * since other KEEP_ALIVE methods are already implemented in the content script and panel script.
- * Code copied from: https://stackoverflow.com/questions/66618136/persistent-service-worker-in-chrome-extension
- */
-const keepAlive = () => setInterval(chrome.runtime.getPlatformInfo, 20e3);
-chrome.runtime.onStartup.addListener(keepAlive);
-keepAlive();
-/* ------------------------------------------------------ */
-
```

Comment:
It would be good to put an explainer in the PR body to explain why we don't need to keep the background script alive using this hack anymore.